### PR TITLE
Master crm lead imp nla

### PR DESCRIPTION
**Pull request by nla-odoo**
- do not show company type on parent creation 
- do not close kanban record create if autocomplete is open

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/views/res_partner_views.xml [hunk 1]
```diff
@@ -80,15 +80,15 @@
                     <field name="avatar_128" invisible="1"/>
                     <field name="image_1920" widget='image' class="oe_avatar" options='{"preview_image": "avatar_128"}'/>
                     <div class="oe_title">
-                        <field name="company_type" options="{'horizontal': true}" widget="radio" groups="base.group_no_one"/>
+                        <field name="company_type" options="{'horizontal': true}" widget="radio" groups="base.group_no_one" invisible="context.get('hide_company_type')"/>
```
Comment (tde-banana-odoo) [inline odoo/addons/base/views/res_partner_views.xml:83]: Would rename to something like ``partner_view_hide_company_type``, to prefix with model + use. Does that sound ok for you ?

File: addons/base_address_city/models/res_partner.py [hunk 2]
```diff
@@ -32,10 +32,12 @@ def _fields_view_get_address(self, arch):
         if doc.xpath("//field[@name='city_id']"):
             return arch
 
+        # only add 'parent_id' invisible field if not already available in arch
+        add_parent_id = not doc.xpath("//field[@name='parent_id']")
```
Comment (tde-banana-odoo) [inline addons/base_address_city/models/res_partner.py:36]: maybe ``parent_id_exist = doc.xpath("//field[@name='parent_id']")`` and use it as inverse condition ?

**Review by tde-banana-odoo — APPROVED** [review]
LGTM, some small changes

**Comment by nla-odoo** [comment]
Hello @tde-banana-odoo 

Changes Done.

Thanks.

**Comment by C3POdoo** [comment]
Dear @nla-odoo,

Thank you for your contribution but we are closing it due to inactivity.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**